src/components/common/layout.js

```diff
@@ -16,15 +16,11 @@ import Header from "./header";
 const globalStyles = css`
   html {
     font-family: Montserrat, Helvetica, Arial;
+    font-size: 16px;
```

Comment:
This will override user agent stylesheet, which is a possible accessibility issue for readers who have set their default font-size larger. Since this is the default, can we set `font-size: 100%` instead?